### PR TITLE
Added sunniness icons on plantings/_thumbnail and show page.

### PR DESCRIPTION
**Pull request by GabrielSandoval**
Hi,

screen shot 2015-07-08 at 10 05 43 am

screen shot 2015-07-08 at 10 11 32 am

I added these icons to reduce the heavy text display on plantings/_thumbnail. I think this would make it more presentable.

Any comments about this?

### CODE REVIEW

File: app/views/plantings/show.html.haml [hunk 1]
```diff
@@ -32,7 +32,7 @@
       %p
         %b 
           = "Sun or shade?: "
-        = "#{display_sunniness(@planting)}"
+        = image_tag("#{@planting.sunniness}.png", :size => "25x25", :title => "#{@planting.sunniness}")
```
Comment (cesy) [inline app/views/plantings/show.html.haml]: What's the reason for skipping the "not specified" here?

File: app/views/plantings/_thumbnail.html.haml [hunk 2]
```diff
@@ -16,7 +16,7 @@
       %dt Finished on:
       %dd= "#{display_finished(planting)}"
       %dt Sun/shade?:
-      %dd= "#{display_sunniness(planting)}"
+      %dd= planting.sunniness.blank? ? "not specified" : image_tag("#{planting.sunniness}.png", :size => "25x25", :title => "#{planting.sunniness}")
```
Comment (cesy) [inline app/views/plantings/_thumbnail.html.haml]: Can we include alt text for screenreaders, please, not just title text?

File: spec/views/plantings/show.html.haml_spec.rb [hunk 3]
```diff
@@ -39,12 +39,6 @@ def create_planting_for(member)
       )
     end
 
-    it "shows the sunniness" do
-      render
-      rendered.should have_content 'Sun or shade?'
-      rendered.should have_content 'sun'
-    end
-
```
Comment (cesy) [inline spec/views/plantings/show.html.haml_spec.rb]: Can we replace this with a check for the alt text and/or for the image, please?

**Comment by cesy** [comment]
I like the idea of more pictures. I've added a few comments for specific details. It may also be worth having the icon alongside the text rather than instead of the text, at least to ease people into the change and get them used to the icons. The icon for shade is very clever, but leaves it inconsistent with the semi-shade one, so I'd appreciate a second opinion on that.

**Comment by maco** [comment]
Perhaps an icon that's a sun with a ? inside would convey "not specified"
rather than switching to text?

**Comment by cesy** [comment]
Yes, that would work.

File: spec/views/plantings/show.html.haml_spec.rb (hunk 3 above)
Comment (GabrielSandoval) [inline spec/views/plantings/show.html.haml_spec.rb]: I can do that, but it says in the file that the test is deprecated. Will it be okay if add some feature tests instead?

**Comment by GabrielSandoval** [comment]
Hi,

A s requested by maco, I created this "not specified" icon.
screen shot 2015-07-09 at 9 13 36 am

I still don't know what to choose between these two for the "semi_shade" icon. What do you guys think?

screen shot 2015-07-09 at 9 13 31 am

**Comment by maco** [comment]
Yes, a feature test is the correct thing to do.

**Pull request closed without merging**